build/build-image/Dockerfile

```diff
@@ -32,10 +32,9 @@ RUN git clone https://github.com/duglin/vlinker.git /vlinker
 ENV PATH=$PATH:/vlinker/bin
 
 # Install the azure client, used to publish svcat binaries
-ENV AZCLI_VERSION=2.0.25
-RUN apt-get update && apt-get install -y python-pip && \
-    rm -rf /var/lib/apt/lists/*
-RUN pip install --disable-pip-version-check --no-cache-dir --upgrade cryptography azure-cli==${AZCLI_VERSION}
+ENV AZCLI_VERSION=v0.1.2
```

Comment:
huh wuh?  2.0.25 == 0.1.2 ?

Reply:
Different cli, different version. The go cli for azure is new, I wrote it the other week, mainly because of problems with installing the python version. 😝

Reply:
Thanks! I bet this will speed up builds a bit.